docker/entrypoint.sh

```diff
@@ -33,15 +33,22 @@ else
   echo "APP_KEY is already set."
 fi
 
-## make sure the db is set up
-echo -e "Migrating Database"
-php artisan migrate --force
-
 echo -e "Optimizing Filament"
 php artisan filament:optimize
 
 # default to caddy not starting
 export SUPERVISORD_CADDY=false
+export PARSED_LE_EMAIL=${LE_EMAIL}
```

Comment:
```suggestion
export PARSED_LE_EMAIL="email ${LE_EMAIL}"
```
Don't you need to prepend it otherwise its just gonna print the raw email in the Caddyfile and screw up the syntax ?